rel-eng/packages/server-attestation-image

```diff
@@ -0,0 +1 @@
+5.0.4 containers/server-attestation-image/
```

Comment:
If you change the image version in the docker file, please, adjust also this. 
From my  POV they should both  be 5.0.0, not  5.0.4